src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1708,6 +1719,11 @@ public Void caseFunctionType(FunctionType type) {
     }
   }
 
+  private boolean isAFunctionPrototypeProp(String propName) {
+    return propName.equals("apply") || propName.equals("call") ||
```

Comment:
This is non-complete, right? Maybe add a comment?

Also, wouldn't an alternative implementation be checking if the declaration location is the Function prototype? That's probably a lot harder, so just asking here.

Also: might be easier to write as a switch statement on propName? Java supports Strings in switches now, AFAIK.